### ISSUE TITLE
feat (ytmusic): `Enable Old Minimized Player`

### ISSUE THREAD

**Issue by ILoveOpenSourceApplications** [revanced/revanced-patches#2633]
### Application

YT Music

### Issue

Recently, YouTube Music removed the ability to swipe down on the minimized player to stop playing music in its latest versions. This feature was highly appreciated by users as it provided a convenient and intuitive way to control playback without needing to expand the player or navigate to a different screen. Removing this feature has caused frustration and inconvenience for many users.

### Patch

A patch that brings back the swipe down gesture to stop playing music on the minimized player. This feature was widely appreciated and integrated well with the overall user experience. By reinstating this functionality, YouTube Music can enhance user satisfaction, improve ease of use, and align with user expectations.

### Motivation

1) Improved User Experience: The swipe down gesture to stop playing music offered a seamless and effortless user experience. It allowed users to quickly and easily pause or stop playback without the need for additional taps or navigating to different screens.

2) Intuitive and Familiar Interaction: The swipe down gesture is a widely recognized and natural action in mobile interfaces. Users have become accustomed to using this gesture for various purposes across different applications. By removing this feature, YouTube Music has deviated from the expected behavior and disrupted the user's muscle memory.

3) Minimized Player Control: The minimized player is designed to provide users with a non-intrusive way to control their music while simultaneously browsing or using other apps. The swipe down gesture to stop playback was a crucial functionality within this context. Users could effortlessly stop music without interrupting their current activity.

4) Accessibility Considerations: The swipe down gesture was especially valuable for users with accessibility needs. It provided a simple and accessible way to pause or stop music without requiring precise tapping or navigation. Removing this feature limits the accessibility options for such users.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by ILoveOpenSourceApplications** [comment]
> 4. Accessibility Considerations: The swipe down gesture was especially valuable for users with accessibility needs. It provided a simple and accessible way to pause or stop music without requiring precise tapping or navigation. Removing this feature limits the accessibility options for such users.

Hello?

**Comment by oSumAtrIX** [comment]
6.04.53

**Comment by ILoveOpenSourceApplications** [comment]
It's removed in the latest versions starting from 6.08+, or at least that's from when I noticed that it's removed. Please consider this as a valid issue.

**Comment by oSumAtrIX** [comment]
I just installed 6.08.50 and the feature persists.

**Comment by ILoveOpenSourceApplications** [comment]
Please restart.

**Comment by ILoveOpenSourceApplications** [comment]
Not for me and I tested it across multiple devices before opening this issue.

**Comment by oSumAtrIX** [comment]
No, the article mentions a slight redesign that adds the cast button:

![image](https://github.com/revanced/revanced-patches/assets/13122796/199a03f5-6fc6-49cb-944b-ea261fea7195)

**Comment by oSumAtrIX** [comment]
Since this is a recent change, wait a couple of weeks. Depending on user feedback, they might bring it back.

**Comment by ILoveOpenSourceApplications** [comment]
That article was from May. The app rollout with this feature is from July. And this change is there in the latest versions.

**Comment by mexon86** [comment]
ym extended has already have this feature just use youtube music extended 👍🏼
